Godeps/Godeps.json

```diff
@@ -40,53 +35,53 @@
 		},
 		{
 			"ImportPath": "github.com/coreos/fleet/client",
-			"Comment": "v0.3.2-976-g96d7f98",
-			"Rev": "96d7f986de72c3fca04b627ddd6578f7206cb7e4"
+			"Comment": "v0.9.0",
```

Comment:
is there a way we could remove all of this duplicate import paths with github.com/coreos/fleet instead, or is this simply autogenerated Godeps?